flab_bada/domain/email/email_repository.py

```diff
@@ -0,0 +1,34 @@
+from flab_bada.database.database import get_redis_db
+from flab_bada.domain import AbstractRepository
+from fastapi import Depends
+from redis import Redis
+
+
+class EmailRedisRepository:
+    def __init__(self, redis_data: Redis = Depends(get_redis_db)):
+        # self.redis = RedisConn().get_session()
+        self.redis = redis_data
+
+    def set_email_secret_data(self, email: str, secret_num: str) -> None:
+        self.redis.set(email, secret_num)
+
+    def get_email_secret_data(self, email: str) -> str:
+        data = self.redis.get(email)
+
+        if data is not None:
+            data = data.decode("utf-8")
+        else:
+            data = ""
+
+        return data
```

Comment:
```suggestion
        if not data:
            return ""
        return data.decode("utf-8")
```
control flow는 위와 같이 에러나 예외 처리를 먼저 해주면 코드 가독성을 높일 수 있습니다